addons/point_of_sale/static/src/js/models.js

```diff
@@ -419,6 +419,12 @@ exports.PosModel = Backbone.Model.extend({
                 return new exports.Product({}, product);
             }));
         },
+    },{
+        model: 'account.cash.rounding',
+        fields: ['name', 'rounding', 'strategy', 'rounding_method'],
```

Comment:
`strategy` and `rounding_method` are not necessary since unused.The JS method only does half-up anyway:

https://github.com/odoo/odoo/blob/c5398bd4bc84a4e2bce16eb22564c87073a28c30/addons/web/static/src/js/core/utils.js#L523